frontend/.eslintrc.json

```
@@ -1,3 +1,27 @@
 {
-  "extends": "next/core-web-vitals"
-}
+  "plugins": [
+    "@typescript-eslint"
+  ],
+  "extends": [
+    "next/core-web-vitals",
+    "prettier",
+    "plugin:@typescript-eslint/recommended"
```

Is this the only plugin that needs to be added? I noticed you added more ESLint packages, so perhaps we should include additional plugins like these:

```suggestion
    "plugin:@typescript-eslint/recommended",
    "plugin:@jsx-a11y/recommended",
    "plugin:react-hooks/recommended",
    "plugin:react/recommended"
```
And so on.

---

I think you are right